tools/replay/ReplayOperationManager.cpp

```diff
@@ -14,7 +14,9 @@
 #include "umpire/Allocator.hpp"
 #include "umpire/strategy/AllocationAdvisor.hpp"
 #include "umpire/strategy/AllocationPrefetcher.hpp"
+#include "umpire/strategy/PoolCoalesceHeuristic.hpp"
 #include "umpire/strategy/SizeLimiter.hpp"
+#include "umpire/strategy/QuickPool.hpp"
```

Comment:
Seeing as you didn't also need to include DynamicPoolList here, is adding QuickPool necessary?